org.fooyin.fooyin.yaml

```diff
@@ -0,0 +1,90 @@
+app-id: org.fooyin.fooyin
```

Comment:
`fooyin.org` doesn't work. (http or https)

Reply:
Not yet. The author just acquired the domain:

```
Domain Name: fooyin.org
Registry Domain ID: f2f11bb40bb9429d923ba43e6c871550-LROR
Registrar WHOIS Server: http://whois.porkbun.com
Registrar URL: https://porkbun.com
Updated Date: 2024-05-18T08:53:02Z
Creation Date: 2024-05-18T08:53:01Z
```

Reply:
Works now 

![image](https://github.com/flathub/flathub/assets/62639087/9919109a-9d5d-47c8-a680-ba81f9914fa4)